Docs/environment_setup/JSON_pipeline_files.md

```diff
@@ -0,0 +1,38 @@
+# Data Factory - Testing Framework
+
+## JSON Files
+
+Ideally, you would integrate your project with a repository such as GIT. Once this is done you will have the JSON files for the Pipelines. If you need extra guidance on that the Microsoft Documentation provides very detailed information how to do it:
+
+Fabric
+[Git integration process - Microsoft Fabric | Microsoft Learn](https://learn.microsoft.com/en-us/fabric/cicd/git-integration/git-integration-process)
+
+
+
+ADF
+[Source control - Azure Data Factory | Microsoft Learn](https://learn.microsoft.com/en-us/azure/data-factory/source-control)
+
+
+
+### Manual steps for testing purposes
+If you want just download one JSON file for a small test purpose and check how the framework here are some ideas on how to do it:
+
+#### ADF 
+
+Refer to the docs if needed: [Azure Data Factory Documentation - Azure Data Factory | Microsoft Learn](https://learn.microsoft.com/en-us/azure/data-factory/)
+
+[Download Azure Data Factory support files - Microsoft Community Hub](https://techcommunity.microsoft.com/t5/azure-data-factory-blog/download-azure-data-factory-support-files/ba-p/1042848)
+
+1. Open your ADF Environment, and open the resource you want the JSON.
+
+2. Click on the action ellipses
+
+3. Click "Download support files" and select the download location in local storage
+
+4. The Zip file contains the JSON Definition
+   
+
+   ![image](https://github.com/LiliamLeme/data-factory-testing-framework/assets/62876278/86c98352-a9f4-4b8e-932e-bc57c2b457bc)
```

Comment:
@LiliamLeme - let's avoid to point to personal repos. I think it is better to create an "Images" folder in the current repo because we all will need to add images of the example pipelines